src/base/utils/version.h

```diff
@@ -167,7 +167,7 @@ namespace Utils
                 throw std::runtime_error ("Incorrect number of version components");
 
             bool ok = false;
-            ComponentsArray res{};
+            ComponentsArray res{0};
```

Comment:
Why not `ComponentsArray res;`?

Reply:
According to some discussions the following can solve this issue with old gcc:
```c++
ComponentsArray res = {};
// or
ComponentsArray res {{}};
```

Reply:
`ComponentsArray res{0};` breaks the coding style (as well as the previous code) and should be fixed: `ComponentsArray res {0};`.
Although, I would still prefer to use one of the options suggested by me above (if it works correctly).